cmake/boards_db.cmake

```diff
@@ -97836,6 +97836,92 @@ target_link_libraries(MAPLEMINI_F103CB_hid INTERFACE
 )
 
 
+# MICROMOD_F405
```

Comment:
Please run cmake/scripts/cmake_updater_hook.py 

This will update properly the cmake part.